src/components/FlexibleForm/Control.js

```diff
@@ -256,6 +256,24 @@ export const Control = memo(props => {
       return <>&nbsp;</>;
     }
 
+    const {
+      childControls: cc,
```

Comment:
Why did you rename this const? In my opinion, `cc` is not a clear name for the variable.

Reply:
The same [here](https://github.com/folio-org/ui-data-import/pull/444/files#diff-6709362c6d184124da81079fe4cef458R250). `ri` is not readable. Even in this PR it took time for me to understand what it means as the line with its assignment is collapsed.